internal/provider/resource_wlan.go

```diff
@@ -137,6 +138,18 @@ func resourceWLAN() *schema.Resource {
 				Optional:    true,
 				Default:     true,
 			},
+			"minimum_data_rate_2g": {
+				Description:  "Enable minimum data rate control for 2G devices",
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"1Mbps", "2Mbps", "5.5Mbps", "6Mbps", "9Mbps", "11Mbps", "12Mbps", "18Mbps", "24Mbps", "36Mbps", "48Mbps", "54Mbps"}, false),
```

Comment:
I think it would be better to rely on server side validation. I am not sure but the accepted values could depend on firmware or controller version.

Reply:
I still think `schema.TypeFloat` would be better here but I could be convinced either way.